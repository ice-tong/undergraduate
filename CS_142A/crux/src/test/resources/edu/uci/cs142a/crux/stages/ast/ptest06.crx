// test redefine predefined

func main() : void {
    ::println();
    var println: int;
    ::println();
}

func printInt(x: int): void {
    ::printInt(x);
}