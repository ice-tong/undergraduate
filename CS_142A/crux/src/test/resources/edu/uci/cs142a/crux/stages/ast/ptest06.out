DeclareSymbolError(9,0)[printInt already exists.]
